deploy/openshift/releases/R2008/manifests/0000000-contrail-08-operator.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: contrail-operator
+  namespace: contrail
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      name: contrail-operator
+  template:
+    metadata:
+      labels:
+        name: contrail-operator
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: name
+                  operator: In
+                  values:
+                  - "contrail-operator"
+            topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      serviceAccountName: contrail-operator
+      hostNetwork: true
+      imagePullSecrets:
+      - name: contrail-registry
+      tolerations:
+        - effect: NoExecute
+          operator: Exists
+        - effect: NoSchedule
+          operator: Exists
+      containers:
+        - name: contrail-operator
+          # Replace this with the built image name
+          image: hub.juniper.net/contrail-nightly/contrail-operator:2008.121
```

Comment:
I think that in the release manifests we should use the `hub.juniper.net/contrail` registry instead of nightly. Greg also changed the official docs to use this registry.

Reply:
Ok, I didn't know about that